devops/PR-Gate.yml

```diff
@@ -1,64 +1,33 @@
-trigger: none
-pr:
-  - master
-
-pool:
-  vmImage: "ubuntu-latest"
+# ADO Pipeline for PRs
 
 variables:
-  artifactFeed: interpret-extensions-feed
-
-steps:
-  - bash: echo "##vso[task.prependpath]$CONDA/bin"
-    displayName: Add conda to PATH
-
-  - template: templates/create-env-template.yml
-
-  - bash: |
-      source activate interp
-      pip install -e ./python
-    displayName: Install interpret-community
-
-  - bash: |
-      source activate interp
-      pip install --upgrade setuptools wheel twine
-      cd ./python
-      python setup.py bdist_wheel
-      cd ..
-    displayName: Create a wheel
+  EnvArtifactStem: 'environment'
+  EnvFileStem: 'environment'
 
-  - task: TwineAuthenticate@0
-    displayName: Configure twine authentication
-    inputs:
-      artifactFeeds: $(artifactFeed)
-
-  - bash: |
-      source activate interp
-      cd ./python
-      twine upload -r $(artifactFeed) --config-file $(PYPIRC_PATH) dist/*
```

Comment:
I'd like to know what this was doing.

Reply:
this is uploading artifacts to the internal feed using twine, so they can be accessed with pip.  Unfortunately I have been having issues with this recently since some recent changes to devops a couple months ago.

Reply:
Certainly, but to what end?

Reply:
Have discussed offline. This might be a useful capability for the future, but it doesn't appear to be working properly right now anyway.